messages/run.js

```diff
@@ -49,9 +48,13 @@ module.exports = {
 	E.g., $ sfdx scanner:run --format xml --target "somefile.js"
 		Evaluates all rules against somefile.js.
 
-	Specifying multiple categories or rulesets is treated as a logical OR.
-		E.g., $ sfdx scanner:run --format xml --target "somefile.js" --category "Design,Best Practices" --ruleset "Braces"
```

Comment:
A mixture of `--category` and `--ruleset` is known not to work. I removed `--ruleset` to avoid giving an example that doesn't work.